app/gui/node-editor.cpp

```diff
@@ -3077,7 +3088,7 @@ editor::show_window() noexcept
                       3,
                       "Fail to initialize example_qss_izhikevich<1>: %s\n",
                       status_string(ret));
-            if (ImGui::MenuItem("Insert example QSS1 seir_lineaire")) 
+           /* if (ImGui::MenuItem("Insert example QSS1 seir_lineaire")) 
```

Comment:
Why you modify this source?

Reply:
I modified this source because after finishing modification on featureSeir branch. I checkout to featureFilter branch and see that the code is showing the old script from before modifying featureSeir branch.  Or should I just leave this alone and uncomment this part?

Reply:
I modified it with a comment because i still see the already corrected and changed script appearing here. But since git uses snapshots , maybe i can just uncomment this and it doesnt affect my already commited featureSeir branch commits

Reply:
I prefer that you don't modify these part of files. This branch must be sync with vle-forge/master and it must treat the new model filter only. Moreover if I accept this change, the second PR will fail to merge this part.

Reply:
I have reverted back to previous state